compressor/management/commands/compress.py

```diff
@@ -211,44 +211,54 @@ def compress(self, log=None, **options):
                       "\n\t".join((t.template_name
                                    for t in compressor_nodes.keys())) + "\n")
 
+        contexts = settings.COMPRESS_OFFLINE_CONTEXT
+        if isinstance(contexts, six.string_types):
+            module, function = settings.COMPRESS_OFFLINE_CONTEXT.rsplit('.', 1)
```

Comment:
Makes sense to use the same approach. The `ValueError` is also caught in `get_mod_func`, although it will lead to an `AttributeError` later due to the empty string for the function name.